Documentation/debugging/using-your-build.md

```diff
@@ -0,0 +1,70 @@
+Using Your Build
+================
+
+This document explains how to use your CoreFX binaries in a .NET Core console app after you've built the repo.
+
+## Overview
+
+You will create a self-contained .NET Core app via `dotnet publish`, which will include stock images of framework assemblies. Then, you will replace some of those assemblies with ones from your CoreFX build.
+
+## Prerequisites
+
+- .NET Core tools - Install at https://www.microsoft.com/net/core
+
+## Steps
+
+0. Create the .NET Core app you want to use with your CoreFX build.
+
+```sh
+mkdir DotnetLab
+cd DotnetLab
+dotnet new
+```
+
+1. Choose a runtime you would like to target your app against.
+
+- A runtime (not to be confused with the [CoreCLR runtime](https://github.com/dotnet/coreclr)) is a specific architecture and OS your .NET app will target.
```

Comment:
Please comment if you think the wording could be improved here/next bullet point.